eos-updater-prepare-volume/eos-updater-prepare-volume

```diff
@@ -173,6 +174,17 @@ class VolumePreparer:
 
         return runtimes
 
+    def _add_autoinstall_flatpaks(self, flatpak_collection_refs):
+        all_actions = self.__run(['/usr/libexec/eos-updater-flatpak-installer', '--mode=print'])
```

Comment:
`libexec` is probably not correct here - `$(libexecdir)` can change depending on whether or not a debian package is being built (for instance, all the libexec stuff in debian packages gets installed into /usr/lib/arch-triple

You probably want to either preprocess this file or have a `config.py.in` where you can substitute the correct `$(libexecdir)` from autotools.

Reply:
That's a good thought. I thought about preprocessing but I didn't think about doing it in a separate Python file. That will be neater.